prekeys.go

```diff
@@ -131,6 +140,54 @@ func (cli *Client) fetchPreKeys(ctx context.Context, users []types.JID) (map[typ
 	return respData, nil
 }
 
+func (cli *Client) FetchPreKeys(ctx context.Context, users []types.JID) (map[types.JID]preKeyResp, error) {
+	return cli.fetchPreKeys(ctx, users)
+}
+
+func (cli *Client) FetchPreKeyTimes(ctx context.Context, users []types.JID) (map[types.JID]int64, error) {
```

Comment:
And what's the point of this? The library handles prekeys internally, users don't need to care about them